sources/osgShadow/ShadowMap.js

```diff
@@ -236,7 +152,14 @@ define( [
 
         // Overridable Visitor so that user can override the visitor to enable disable
         // in its own shadowmap implementation
-        this._removeNodesNeverCastingVisitor = new RemoveNodesNeverCastingVisitor( this._castsShadowTraversalMask );
+        // settings.userShadowCasterVisitor:
+        // - undefined means using default
+        // - false means no removal visitor needed
+        // - otherwiser must be an instance of a class inherited from shadowCaster
+        if ( settings.userShadowCasterVisitor !== false ) {
```

Comment:
As discussed before, settings is considerer a dictionnary not an object.
But can add an accessor if feeling is strong on that one.